### PR TITLE
issue-145 - User Story page is created

### PR DESCRIPTION
**Pull request by suhhee1011**
## PR Description 
This Pull request is about fixed issue #145  
- To fix the issue, What I changed are:
     -  Created a stories module to display user stories.
         - two user stories per row.(Tried to create 3 in a row, but it becomes so small and hard to read)
     -  Added path in the router to let users can access the page through the menu bar
     -  Created userData.js to help test the User Story Page.
  
     

### How to get to the page

1. Open index.html locally.
2. Click User story in the menu bar

## Result - User Story Page
image

### CODE REVIEW

File: src/components/layout/UserCard.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  overflow: hidden;
+}
+@media (max-width: 990px) {
+  .card .desc {
+    font-size: 1em;
+  }
+  .story_Section .story {
+    font-size: 0.8em;
+  }
+}
```
Comment (redxzeta) [inline src/components/layout/UserCard.css:112]: nice changes but it interferes with the pet card designs. maybe try renaming them

File: src/components/layout/PetCard.css [hunk 2]
```diff
@@ -48,7 +48,8 @@
 
 .card__img {
   object-fit: cover;
-  height: 300px;
+  height: 50%;
+  width: 50%;
```
Comment (redxzeta) [inline src/components/layout/PetCard.css:52]: lets not change this css maybe just create a new one specifically for the user story cards

File: src/components/layout/PetCard.css [hunk 3]
```diff
@@ -7,7 +7,7 @@
   padding-top: 10px;
   -webkit-box-shadow: 0px 10px 13px -7px #000000,
     5px 5px 15px 5px rgba(0, 0, 0, 0);
-  width: 20rem;
+  width: 30rem;
```
Comment (redxzeta) [inline src/components/layout/PetCard.css:10]: lets not change this css maybe just create a new one specifically for the user story cards

**Review by redxzeta — CHANGES_REQUESTED** [review]
nice changes I really like the way it looks

Some of the styles interfere with the pet design cards

image

and some fixes for mobile responsiveness

image

File: src/components/layout/UserCard.css (hunk 1 above)
Comment (suhhee1011) [reply src/components/layout/UserCard.css:112]: solved!

File: src/components/layout/PetCard.css (hunk 3 above)
Comment (suhhee1011) [reply src/components/layout/PetCard.css:10]: solved!

**Comment by suhhee1011** [comment]
Hi @redxzeta I fixed responsive and PetCard.css based on your comment

File: src/components/layout/PetCard.css (hunk 2 above)
Comment (suhhee1011) [reply src/components/layout/PetCard.css]: solved!

**Review by redxzeta — APPROVED** [review]

**Pull request merged**